core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -358,11 +363,11 @@ object AdminUtils extends Logging {
     zkUtils.zkClient.exists(getTopicPath(topic))
 
   def getBrokersAndRackInfo(zkUtils: ZkUtils, rackAwareMode: RackAwareMode = RackAwareMode.Enforced,
-                            brokerList: Seq[Int] = Seq()): (Seq[Int], Map[Int, String]) = {
+                            brokerList: Option[Seq[Int]] = None): (Seq[Int], Map[Int, String]) = {
```

Comment:
This is to make the semantics clear. Using empty list to mean "all" is a common source of bugs and we have been bitten by that in Kafka a few times.